libraries/core-react/src/hooks/useEscapePress.ts

```diff
@@ -0,0 +1,23 @@
+import { useEffect } from 'react'
+
+export const useEscapePress = (callback: (e: KeyboardEvent) => void): void => {
```

Comment:
I suggest changing the signature of this hook so its abit more applicable for other keys (not just "Escape"). 

If you change it to be more generic such as `useGlobalKeyPress` and also take in a parameter for which Key (as first parameter). Then it will be easier for us to re-use this hook for other use-case and keys.

Since we are using typescript you can also make an [enum type](https://www.typescriptlang.org/docs/handbook/enums.html) for KEY so that we can get type-safety for [keycodes](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values) on the provided parameter for which key.

example of enum:
```typescript 
enum KEY = {
Escape = 'Escape' 
}
```